src/data/codingcontracttypes.ts

```diff
@@ -85,10 +91,9 @@ export const codingContractTypesMetadata: ICodingContractTypeMetadata[] = [
 
       return (n === 1 ? fac - 1 : n) === parseInt(ans, 10);
     },
-  },
+  } as CodingContractType<number>,
```

Comment:
Not sure if this *works*, but `satisfies CodingContractType<number>` should still check for the object to match the specified form, instead of just saying it can't be coerced?

Reply:
Good call.